src/index.ts

```diff
@@ -93,12 +103,21 @@ async function initVueRouterGuard(
     return;
   }
 
-  // Flatten routes name
-  ignoredViews = ignoredViews.map((view) => view.toLowerCase());
+  // Normalize routes name
+  const normalizedIgnoredViews: (string | RegExp)[] = ignoredViews.map((view) =>
+    view instanceof RegExp ? view : view.toLowerCase()
+  );
+  const shouldIgnoredView = (routeName: string): boolean => {
+    const found = normalizedIgnoredViews.find((ignoredView) =>
+      ignoredView instanceof RegExp ? ignoredView.test(routeName) : routeName.toLowerCase() === ignoredView
+    );
+
+    return !!found;
```

Comment:
I think this can be shortened by using `.some` instead of `.find`

Reply:
your are right, I'm going to fix this!